extensions/measurement-tracking/src/getPanelModule.tsx

```diff
@@ -11,9 +15,10 @@ function getPanelModule({
   commandsManager,
   extensionManager,
   servicesManager,
-}) {
+}): Types.Panel[] {
   return [
     {
+      id: '@ohif/extension-measurement-tracking.panelModule.seriesList',
```

Comment:
I'm not sure why we are adding id here. Extension manager dynaicly will create this id from the `name` here https://github.com/OHIF/Viewers/blob/v3-stable/platform/core/src/extensions/ExtensionManager.ts#L374

Reply:
Yes this bothered me a bit too. Well the `id` needs to be added to each individual panel module (i.e. each tab) so that `SidePanel` can register activation listeners for each of its tabs. I am about to push code where the `ExtensionManager` now sets the id for the various extension modules including the panel modules. I am not 100% certain this is the best way and if not I am sure you will be able to help me find something better. 😊